src/misc.jl

```diff
@@ -1,13 +1,13 @@
 #Is this usable data?
-isconcrete{T<:Number}(x::T) = !isna(x) && isfinite(x)
+isconcrete{T<:Number}(x::T) = !ismissing(x) && isfinite(x)
```

Comment:
I guess it should be something like:
```julia
isconcrete(::Missing) = false
isconcrete(x::Number) = isfinite(x)
```

Reply:
Good catch. This should fix https://github.com/GiovineItalia/Gadfly.jl/issues/1083 too, I think